frontend/src/pages/TenantsPage.tsx

```
@@ -56,6 +57,27 @@ export const TenantsPage: React.FC = () => {
+    return <Spin tip="Loading tenants..." />;
+  }
+
+  if (tenants.length === 0 && !loading) {
+    return (
+      <Card>
+        <div style={{ textAlign: 'center', padding: 48 }}>
+          <Typography.Title level={4}>No Tenants Found</Typography.Title>
+          <Typography.Text type="secondary">
+            Create your first tenant to get started.
+          </Typography.Text>
+          <Button type="primary" onClick={handleNewTenant} style={{ marginTop: 16 }}>
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)